app/controllers/goals_controller.rb

```diff
@@ -6,8 +6,28 @@ def index
   end
 
   def create
-    Goal.create_goal(name: params[:name])
+    Goal.create_goal(goal_params)
     render json:{}, status: :created
   end
 
+  def update
```

Comment:
Let's refactor the code below by extracting an `update_goal` method to the goal model (as we have done with `create_goal` and `get_goals`.